2021q3/pot.adoc

```diff
@@ -0,0 +1,31 @@
+=== Containers & FreeBSD: Pot, Potluck & Potman
+
+Links: +
+link:https://pot.pizzamig.dev[Pot on github] URL: link:https://github.com/pizzamig/pot[https://github.com/pizzamig/pot] +
+link:https://potluck.honeyguide.net/[Potluck Repository & Project] URL: link:https://potluck.honeyguide.net/[https://potluck.honeyguide.net/] +
+link:https://github.com/hny-gd/potluck[Potluck on github] URL: link:https://github.com/hny-gd/potluck[https://github.com/hny-gd/potluck] +
+link:https://github.com/grembo/potman[Potman on github] URL: link:https://github.com/grembo/potman[https://github.com/grembo/potman]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Stephan Lichtenauer (Potluck) <sl@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@FreeBSD.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+In the last quarter, a new release link:https://github.com/pizzamig/pot/releases/tag/0.13.0[0.13.0] with a number of major new features has been released. +
+link:https://github.com/pizzamig/pot/issues/148[Layered images] allow a split into a foundation image component that only changes seldomly and that e.g. contains the basic jail and packages and a "leaf" image component on top with potentially small additions that might change more frequently, like software built in a CI pipeline. Since it is not the complete image that has to be rebuilt each time, image creation and distribution can be sped up immensely. +
+Also a link:https://github.com/pizzamig/pot/issues/162[copy-out command] has been included where the container state that was initialised from within the container can be persisted and reinjected into additional containers afterwards.   
+
+Potluck aims to be to FreeBSD and Pot what Dockerhub is to Linux and Docker: A repository of Pot flavours and complete container images for usage with Pot and in many cases Nomad.
+
+Here we had a busy quarter. Not only did we commit a number of new images like link:https://potluck.honeyguide.net/blog/nextcloud-nginx-nomad/[Nextcloud] which can be deployed via Nomad, also the images that can be used to build the foundation of a virtual DC (link:https://potluck.honeyguide.net/blog/nomad-server/[Nomad],  link:https://potluck.honeyguide.net/blog/consul/[Consul] and link:https://potluck.honeyguide.net/blog/vault/[Vault]) themselves have received major updates.
```

Comment:
s/also the images that can be used/but the images used/
"DC"? data center?
Spurious space before "link:https://potluck.honeyguide.net/blog/consul/[Consul]"